_posts/learn-github.md

```diff
@@ -0,0 +1,17 @@
+---
+layout: chapter
+title: Learn Git
+order: 1
+category: 3. Code Many Projects
```

Comment:
Shouldn't this be inserted somewhere in the chunk where we introduce Git/Github? (2. Publish Code)

It also might require editing the intro/outro for the chapters it gets put between, to make the curriculum sound continuous.